grafana-plugin/src/containers/PluginConfigPage/PluginConfigPage.tsx

```diff
@@ -196,16 +186,24 @@ const PluginConfigPage: FC<OnCallPluginConfigPageProps> = ({
     content = (
       <>
         <StatusMessageBlock text={pluginConnectionCheckError || pluginResetError} />
-        <RemoveConfigButton />
+        <HorizontalGroup>
+          <Button variant="primary" onClick={triggerDataSyncWithOnCall} size="md">
+            Retry Sync
+          </Button>
+          {licenseType === GRAFANA_LICENSE_OSS ? <RemoveConfigButton /> : null}
+        </HorizontalGroup>
       </>
     );
   } else if (syncError) {
     content = (
       <>
         <StatusMessageBlock text={syncError} />
-        <Button variant="primary" onClick={triggerDataSyncWithOnCall} size="md">
-          Retry Sync
-        </Button>
+        <HorizontalGroup>
+          <Button variant="primary" onClick={triggerDataSyncWithOnCall} size="md">
+            Retry Sync
+          </Button>
+          {licenseType === GRAFANA_LICENSE_OSS ? <RemoveConfigButton /> : null}
```

Comment:
_nit_ consider deduplicating this block into a component that can be reused:
```tsx
const ReconfigurePluginButtons = () => (
  <HorizontalGroup>
    <Button variant="primary" onClick={triggerDataSyncWithOnCall} size="md">
      Retry Sync
    </Button>
    {licenseType === GRAFANA_LICENSE_OSS ? <RemoveConfigButton /> : null}
  </HorizontalGroup>
);
```
(probably nested inside of the `PluginConfigPage` function so that it has access to `licenseType` and `triggerDataSyncWithOnCall`)